src/apps/loan-list/materials/stackable-material.tsx

```diff
@@ -70,121 +82,126 @@ const StackableMaterial: FC<StackableMaterialProps & MaterialProps> = ({
+                {creators &&
+                  getAuthorNames(
+                    creators,
+                    t("loanListMaterialByAuthorText"),
+                    t("loanListMaterialAndAuthorText")
+                  )}
+                {year?.year && <> ({year.year})</>}
+              </p>
+            </div>
+            {additionalMaterials > 0 && (
+              <>
+                <div
+                  className="list-reservation__hidden-explanation"
+                  id="materials-modal-desktop-text"
+                >
+                  {t("loanListMaterialsModalDesktopText")}
+                </div>
+                <button
+                  type="button"
+                  onClick={(e) => openDueDateModalCallBack(e)}
+                  aria-describedby="materials-modal-desktop-text"
+                  id="test-more-materials"
+                  className="list-reservation__note-desktop"
+                >
+                  + {additionalMaterials} {t("loanListMaterialsDesktopText")}
+                </button>
+              </>
+            )}
+            {dueDate && materialIsOverdue(dueDate) && (
+              <a
+                href="todo"
+                className="list-reservation__note-desktop color-signal-alert"
               >
-                + {additionalMaterials} {t("loanListMaterialsDesktopText")}
-              </button>
-            </>
-          )}
-          {materialIsOverdue(dueDate) && (
-            <a
-              href="todo"
-              className="list-reservation__note-desktop text-small-caption color-signal-alert"
-            >
-              {t("loanListLateFeeDesktopText")}
-            </a>
-          )}
+                {t("loanListLateFeeDesktopText")}
+              </a>
+            )}
+          </div>
         </div>
-      </div>
-      <div>
-        <div className="list-reservation__status">
-          <StatusCircle loanDate={loanDate} dueDate={dueDate} />
-          <div>
-            <div className="list-reservation__deadline">
-              <StatusBadge
-                dueDate={dueDate}
-                dangerText={t("loanListStatusBadgeDangerText")}
-                warningText={t("loanListStatusBadgeWarningText")}
-              />
-              <p className="text-small-caption" id="due-date">
-                {t("loanListToBeDeliveredText")} {formatDate(dueDate)}
-              </p>
-              {additionalMaterials > 0 && (
-                <>
-                  <div
-                    className="list-reservation__hidden-explanation"
-                    id="materials-modal-text"
-                  >
-                    {t("loanListMaterialsModalMobileText")}
-                  </div>
-                  <button
-                    type="button"
-                    aria-describedby="materials-modal-text"
-                    className="list-reservation__note-mobile text-small-caption color-secondary-gray"
+        {dueDate && loanDate && (
```

Comment:
You could consider creating a new variable that is called something that describes  what `dueDate && loanDate` is for making it more clear what is being checked. Or write a comment about it :)